tests/58-live-tsync_notify.c

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
The tests look reasonable to me.  Did you run them by hand?  TravisCI skipped them since its kernel doesn't support this feature yet.

Reply:
Yes, my dev system has a new enough kernel:
```
% uname -r; ./regression -T live -b 58-live-tsync_notify
5.7.6-arch1-1
=============== Sat 27 Jun 2020 05:18:12 PM EDT ===============
Regression Test Report ("regression -T live -b 58-live-tsync_notify")
 batch name: 58-live-tsync_notify
 test mode:  c
 test type:  live
Test 58-live-tsync_notify%%001-00001 result:   SUCCESS
 batch name: 58-live-tsync_notify
 test mode:  python
 test type:  live
Test 58-live-tsync_notify%%001-00001 result:   SUCCESS
Regression Test Summary
 tests run: 2
 tests skipped: 0
 tests passed: 2
 tests failed: 0
 tests errored: 0
============================================================
```